rest/oidc_test_provider_test.go

```diff
@@ -169,6 +232,140 @@ func TestProviderOIDCAuthWithTlsSkipVerifyDisabled(t *testing.T) {
 	require.NoError(t, response.Body.Close(), "Error closing response body")
 }
 
+func TestOpenIDConnectTestProviderWithRealWorldToken(t *testing.T) {
+	tests := []struct {
+		name          string              // Unit test name.
+		idTokenFormat identityTokenFormat // Specific token format.
+		algExpected   string              // Identifies the cryptographic algorithm used to secure the JWS.
```

Comment:
Do these tests really need to verify alg/kid/typ/x5t/ver?  I would have thought the goal of the test was just to validate successful authentication - what's the reason to validate the rest of these properties?

Reply:
Reviewed w/ @sarathkumarsivan - test is validating test provider token format selection as well as validating authentication.